### DISCUSSION TITLE
Comment pane feature from Slide for Reddit.

### DISCUSSION THREAD

**Discussion by RandomIrrelevantUsername** [Docile-Alligator/Infinity-For-Reddit#985] [Ideas]
Comment pane saves our current comment position after going back via back gesture, back button press or swiping to go back. This is very useful if we are reading a comment that's far down the page and we accidentally go back.

**Comment by Wladefant** [comment]
@RandomIrrelevantUsername definiyely worth implementing.

Everyone who is interested in this feature please give a thumbs up to prioritize this feature.
